### ISSUE TITLE
Add benchmarks result to the readme

### ISSUE THREAD

**Issue by ChShersh** [kowainik/co-log#76]
Ideally I would like to see pure function that takes list of pairs -- benchmark name and result and generates markdown table which we can then insert to the `README` easily in case we want to update benchmark results.

**Comment by C37H40O9** [comment]
Something like this?

```haskell
import Data.List (intercalate)
import Data.Monoid ((<>))

genTable pairs = intercalate "\n" $ header <> rows
    where
      header = intercalate " | " <$> [["Benchmark", "Result"], ["---","---:"]]
      rows = fmap (\(b,r) -> b <> " | " <> show r) pairs
```
For `[("B1",1),("B2",12),("B3",123)]` it produce
```markdown
Benchmark | Result
--- | ---:
B1 | 1
B2 | 12
B3 | 123
```

Benchmark | Result
--- | ---:
B1 | 1
B2 | 12
B3 | 123

**Comment by C37H40O9** [comment]
Version with aligning:
```haskell
import Data.List (intercalate)
import Data.Semigroup((<>),Max(..))
import Data.Bifunctor (bimap, second)
import Data.Coerce (coerce)

genTable pairs = intercalate "\n" rows
  where
    pairs' = ("Benchmark", "Result") : ("---","---:") : fmap (second show) pairs
    f = (Max).length
    (fMax,sMax) = coerce $ foldMap (bimap f f) $ fmap (second show) pairs'
    rows = fmap (\(b,r) -> b <> replicate (fMax - (length b)) ' ' <> " | " 
                             <> replicate (sMax - (length r)) ' ' <> r) pairs'
```
produce
```
Benchmark             |   Result
---                   |     ---:
VeryLongBenchmarkName |        1
ShortBench            |       12
MediumBenchmark       |      123

```
Benchmark             |   Result
---                   |     ---:
VeryLongBenchmarkName |        1
ShortBench            |       12
MediumBenchmark       |      123

**Comment by ChShersh** [comment]
@C37H40O9 this looks nice! You're welcome to open PR if you want 🙂

**Comment by C37H40O9** [comment]
Cool! My (first) PR waiting for review :-)

**Comment by ChShersh** [comment]
The function for formatting is implemented in #90. I will refactor code of benchmarks a little bit (by adding description to benchmarks, etc.) and then I will insert the results into README.

**Comment by ChShersh** [comment]
Closed by #92
